tests/Silex/Tests/Provider/ValidatorServiceProviderTest.php

```diff
@@ -55,6 +56,21 @@ public function testRegisterWithCustomValidators()
         return $app;
     }
 
+    public function testRegisterWithAnnotationLoader()
```

Comment:
They test if there is some kind of error while registering the service.
Just like many other tests do around in the framework.